x/or/settlement/keeper/keeper.go

```diff
@@ -0,0 +1,46 @@
+package keeper
+
+import (
+	"fmt"
+
+	"github.com/Finschia/ostracon/libs/log"
+
+	"github.com/Finschia/finschia-sdk/codec"
+	sdk "github.com/Finschia/finschia-sdk/types"
+	"github.com/Finschia/finschia-sdk/x/or/settlement/types"
+	paramtypes "github.com/Finschia/finschia-sdk/x/params/types"
+)
+
+type (
+	Keeper struct {
+		cdc        codec.BinaryCodec
+		storeKey   sdk.StoreKey
+		memKey     sdk.StoreKey
+		paramstore paramtypes.Subspace
```

Comment:
What about use local KVStore for module params instead of using deprecated param store?